### PR TITLE
kfserving: Add support for serving tensorflow models from mlflow

### PR DESCRIPTION
**Pull request by flaviodsr**

### CODE REVIEW

File: images/inference-services/kfserving/run.sh [hunk 1]
```diff
@@ -14,15 +14,40 @@ else
...
 
-# kfserving expects the model file as model.joblib however mlflow
-# saves the model as model.pkl, so if there is no model.joblib,
-# create a copy named model.joblib from model.pkl
 mc alias set minio http://mlflow-minio:9000 ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY}
 model_bucket="minio${FUSEML_MODEL//s3:\//}"
-if ! mc ls ${model_bucket} | grep -q "model.joblib"; then
-    mc cp ${model_bucket}/model.pkl ${model_bucket}/model.joblib
+
+export PREDICTOR=${FUSEML_PREDICTOR}
```
Comment (jsuchome) [inline images/inference-services/kfserving/run.sh:20]: Who sets `FUSEML_PREDICTOR` value?

Comment (flaviodsr) [reply images/inference-services/kfserving/run.sh:20]: it is one of the inputs from the `predictor` step

Comment (jsuchome) [reply images/inference-services/kfserving/run.sh:20]: Ah, and the env variable (with `FUSEML_` prefix is created automatically from all the inputs in the workflow, right? I vaguely remember now, but do we have it documented somewhere?

Comment (flaviodsr) [reply images/inference-services/kfserving/run.sh:20]: correct, afaik there is not documentation about workflows yet

Comment (jsuchome) [reply images/inference-services/kfserving/run.sh:20]: OK, such discussion is out of the scope of this PR, it just shows how much hidden information we already have at this time...

**Review by jsuchome — APPROVED** [review]

File: images/inference-services/kfserving/run.sh (hunk 1 above)
Comment (flaviodsr) [reply images/inference-services/kfserving/run.sh:20]: Indeed, just like anything that lacks documentation.

**Review by jsuchome — APPROVED** [review]

**Pull request merged**